sklearn/metrics/_plot/roc_curve.py

```diff
@@ -164,7 +267,13 @@ def plot(
             default_chance_level_line_kw, chance_level_kw
         )
 
-        (self.line_,) = self.ax_.plot(self.fpr, self.tpr, **line_kwargs)
+        self.line_ = []
+        for fpr, tpr, line_kw in zip(fpr, tpr, curve_kwargs):
+            self.line_.extend(self.ax_.plot(fpr, tpr, **line_kw))
```

Comment:
`self.ax_.plot(...)` will return a list. So if I'm not mistaken, we should have `self.line_` to be a list of list of Line2D. It is not inline with the documentation. I'm wondering if we should not have something like:


```suggestion
            self.line_.extend(self.ax_.plot(fpr, tpr, **line_kw)[0])
```

Reply:
But I might miss something. I should check if we have a test checking the attribute.